src/content/ui-api/web/organizations/access-control/index.md

```diff
@@ -0,0 +1,171 @@
+---
+linkTitle: Access control
+title: Access control for organizations in the web user interface
+description: Learn how to use our web UI for organization RBAC to quickly associate service accounts with organizations.
+weight: 20
+menu:
+  main:
+    identifier: orgs-accesscontrol
+    parent: web-orgs
+owner:
+  - https://github.com/orgs/giantswarm/teams/biscuit
+user_questions:
+  - How can I inspect roles and permissions bound to an org-namespace?
+  - How can I create a service account in the web interface?
+  - How can I bind a service account to an org-namespace in the web interface?
+last_review_date: 2021-06-16
+---
+
+# Access control for organizations in the web user interface
+
+Learn how to quickly grant and revoke access to an organization's resources via the web user interface, using Kubernetes role based access control (RBAC) mechanisms.
+
+{{% organization_mgmt_mapi_disclaimer %}}
+
+Granting access means that the users (when associating users or groups) or programs (when using service accounts) affected will be able to access resources via the Management API, including via the web user interface itself.
+
+The access control UI in the web interface provides quick access to a focused set of RBAC resources, with the goal to make it easy for you to give someone &ndash; be it individuals or groups &ndash; or service accounts access to resources in an organization's namespace.
```

Comment:
```suggestion
The access control UI in the web interface provides quick access to a focused set of RBAC resources, with the goal to make it easy for you to give individuals, groups, or service accounts access to resources in an organization's namespace.
```

does that read a little easier? i don't love the current construction, but it's hard to find something better